users_ldap_groups/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "Groups assignment",
-    "version": "1.2",
+    "version": "7.0.1.2.1",
```

Comment:
I think 7.0 doesn't support preprending major version, so I think the module version as is will be 7.0.7.0.1.2.1